[validate, error-code] - fix bugs (#3504)
demisto/demisto-sdk: CHANGELOG.md

* Removed the use of chunks when calculating content relationships.
* Fixed an issue where the url regex in the **validate** command was wrong.
* Fixed an issue where **pre-commit** command failed when using global environment.

> * Fixed an issue where the **validate** command would fail on external repos when trying to ignore `BA102`.

Reviewer: ```suggestion
* Fixed an issue where **validate** would fail in external repos when trying to ignore `BA102`.
```

> * Fixed an issue where the **error-code** command failed on some error codes.

Reviewer: ```suggestion
* Fixed an issue where **error-code** failed on some error codes.
```
